cmd/node/main.go

```diff
@@ -126,7 +129,13 @@ func startNodeRunner(c *cli.Context, log logger.Logger, version string) error {
 		return errRunner
 	}
 
-	err = nodeRunner.Start()
+	//TODO: Activate the following code for sovereign chain and also use the resulted instance, instead the original one, to call the method Start() below
```

Comment:
I would not take this approach of having to change the code manually.
Reason: commented code can not be maintained as it does not raise compilation errors and it will just rot. 
Suggestion: let's have a fresh new main.go file. I'm sure we will need some new custom code for the sovereign chain nodes in the future.

Reply:
Refactored in PR #4659